code/game/machinery/antiquemattersynth.dm

```diff
@@ -0,0 +1,163 @@
+
+var/global/list/synth_designs = list(
+list("category" = "resources", "name" = "sulfuric acid bottle", "path" = /obj/item/weapon/reagent_containers/glass/bottle/sacid, "cost" = 100),
+list("category" = "resources", "name" = "20 iron", "path" = /obj/item/stack/sheet/metal/bigstack, "cost" = 100),
+list("category" = "resources", "name" = "20 glass", "path" = /obj/item/stack/sheet/glass/glass/bigstack, "cost" = 100),
+list("category" = "resources", "name" = "20 plasteel", "path" = /obj/item/stack/sheet/plasteel/bigstack, "cost" = 100),
+list("category" = "resources", "name" = "20 plasmaglass", "path" = /obj/item/stack/sheet/glass/plasmaglass/bigstack, "cost" = 100),
+list("category" = "resources", "name" = "20 wood", "path" = /obj/item/stack/sheet/wood/bigstack, "cost" = 100),
+list("category" = "resources", "name" = "20 plastic", "path" = /obj/item/stack/sheet/mineral/plastic/bigstack, "cost" = 200),
+list("category" = "resources", "name" = "aluminum foam grenade", "path" = /obj/item/weapon/grenade/chem_grenade/metalfoam, "cost" = 100),
+list("category" = "resources", "name" = "iron foam grenade", "path" = /obj/item/weapon/grenade/chem_grenade/ironfoam, "cost" = 100),
+list("category" = "resources", "name" = "camera assembly", "path" = /obj/item/weapon/camera_assembly, "cost" = 100),
```

Comment:
this is a weird one. an autolathe can make 4 of them with one sheet of metal and glass, yet this one costs as much as synthesizing 10 iron and 10 glass
I'd just remove it

Reply:
it's an antique, it's not supposed to be efficient